test-app/ember-cli-build.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function (defaults) {
+  const app = new EmberApp(defaults, {
+    // Add options here
+    autoImport: {
+      watchDependencies: ['ember-container-query'],
+    },
+  });
```

Comment:
These lines allow the `test-app` package to rebuild automatically when the `addon` package is changed.

https://github.com/embroider-build/addon-blueprint/blob/v1.4.1/files/test-app-overrides/ember-cli-build.js#L6-L10